sys/pre-commit-indent

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# A hook script to verify what is about to be committed follows the coding
+# style. Called by "git commit" with no arguments. The hook should exit with
+# non-zero status after issuing an appropriate message if it wants to stop the
+# commit.
+#
+# To enable this hook, move it to ".git/hooks/pre-commit".
+
+tmpfile="$(mktemp)"
+unstaged_diff="$(mktemp)"
+git diff > "${unstaged_diff}"
```

Comment:
can be "", chk if mktemp is not failing and tmpfile and unsaged_diff are not. ""